src/common/pluginloader.js

```diff
@@ -110,6 +116,26 @@ function findCordovaPath() {
     return path;
 }
 
+exports.setModuleInstrumenter = function(moduleID, instrumenter) {
+    if (typeof moduleID !== "string" || typeof instrumenter !== "function") {
+        // ignore
+        return;
+    }
+    moduleInstrumenters[moduleID] = instrumenter;
+}
+
+// Recovers the information from the plugin list metadata and associates each
+// module with its corresponding plugin
```

Comment:
this will not be needed after apache/cordova-lib#322 gets into a released cordova-cli version.